haiku-misc/vcardpeople/vcardpeople-1.0.1.recipe

```diff
@@ -0,0 +1,62 @@
+SUMMARY="Utilities to convert between VCard and BeOS 'Person' files"
+DESCRIPTION="The VCard format is an open standard used for storing and \
+interchanging personal (\"Business Card\") data by applications on many \
+platforms. The two small programs in this package will convert between this \
+format and BeOS 'Person' files, making it possible to share contact data \
+between BeOS and other platforms."
+HOMEPAGE="http://goodeveca.net/beos/"
+COPYRIGHT="2002 Peter J. Goodeve"
+LICENSE="Public Domain"
```

Comment:
This port is not unlicensed but under a custom license

From `vcard2people.cpp`
>```c
>// Copyright 2002 Peter J. Goodeve
>/***************************************************************
>You are permitted to use this code for any purpose, provided that
>the original author is acknowledged.
>No warranty as to fitness for any particular purpose, nor responsibility
>for any consequential damage, is expressed or implied.
>****************************************************************/
>```

```diff
-LICENSE="Public Domain"
+LICENSE="VCard to Person"

Reply:
Will haikuporter complain on this?

Reply:
It will complain if it cannot find a file called "VCard to Person" in `/system/data/licenses/` or in the `haikuports/haiku-misc/vcardpeople/licenses/`.
This means you need to create a `licenses` subdirectory (next the recipe) and place in there a file with the text of the license.
(Popular licenses are shipped in Haiku, so most packages don't need to have that licenses directory.)